src/components/Action/Action.vue

```diff
@@ -0,0 +1,93 @@
+<!--
+  - @copyright Copyright (c) 2018 John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @author John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<!-- if only one action, check if we need to bind to click or not -->
+	<a :tabindex="actions.length === 1" :href="actions.length === 1 && actions[0].href ? actions[0].href : '#'"
+		:class="[actions.length === 1 ? `${actions[0].icon} action-item--single` : 'action-item--multiple']"
+		v-on="actions.length === 1 && actions[0].action ? { click: actions[0].action } : {}">
+
+		<!-- If more than one action, create a popovermenu -->
+		<template v-if="actions.length > 1">
+			<span v-click-outside="closeMenu" tabindex="1" class="action-item__menutoggle icon-more"
+				@click="toggleMenu" />
+			<span :class="{ 'open': opened }" class="action-item__menu popovermenu">
```

Comment:
Is `action-item__menu` a real class? What's that used for?

Reply:
Will be used for scoped design